crates/iota-framework/packages/iota-system/sources/iota_system_state_inner.move

```diff
@@ -734,9 +735,9 @@ module iota_system::iota_system_state_inner {
                 storage_rebate: storage_rebate_amount,
                 storage_fund_balance: self.storage_fund.total_balance(),
                 total_gas_fees: computation_charge,
-                total_stake_rewards_distributed: total_validator_rewards_distributed,
+                total_stake_rewards_distributed,
                 burnt_tokens_amount,
-                minted_tokens_amount
+                minted_tokens_amount: validator_target_reward,
```

Comment:
Can we use the `total_validator_rewards_amount_before_distribution` variable here or create a new `minted_tokens_amount` variable which will contain the `total_validator_rewards` value?
In this case, it reflects the `match_computation_reward_to_target_reward` logic.

Reply:
Fair point, I actually decided to address this by getting the minted and burned amounts directly from the match_computation_reward_to_target_reward as it was before